docker/docker-compose.yml

```diff
@@ -28,10 +28,11 @@ services:
       - ../:/usr/local/src/timesketch/
 
   elasticsearch:
+
+    environment:
+      - TAKE_FILE_OWNERSHIP=1
```

Comment:
After reviewing the repo for elasticsearch Docker I found the [entrypoint](https://github.com/elastic/elasticsearch/blob/master/distribution/docker/src/docker/bin/docker-entrypoint.sh) which referenced an environment variable at line 95 used to chmod the data directory to the correct uid,gid values.  

Adding that enviroment variable did the trick and now /var/lib/elasticsearch does not need to be precreated and chown 1000:1000 does not need to be run on it.

Reply:
Sounds good!